string class_name
float32 score
int32 x_min
int32 x_Max
int32 y_min
int32 y_Max